backup/testing/montagu-deploy.json

```diff
@@ -15,6 +15,6 @@
     "require_clean_git": false,
     "add_test_user": false,
     "enable_db_replication": true,
-    "password_group": "production",
+    "password_group": "fake",
```

Comment:
I believe this does need to be production, but I can't remember why!

Reply:
yes you're right, my bad